src/components/popover/popover.tsx

```diff
@@ -176,6 +176,10 @@ export interface EuiPopoverProps {
    * Use case is typically limited to an accompanying `EuiBeacon`
    */
   arrowChildren?: ReactNode;
+  /**
+   * IDs used in ARIA and labels
+   */
+  ariaLabelledId?: string;
```

Comment:
Nice! This is close to what we need!

Let's use the actual `aria-*` attribute names (and add `aria-label` as an option) as they're not needed on the container anyways then this will work more closely to how people might expect without having to refer to docs.

```suggestion
  /**
   * Provide a name to the popover
   **
  'aria-label'?: string;
  /**
   * Alternative option to `aria-label` that takes an `id`
   * Use if Popover has a visible title
   **
  'aria-labelledby'?: string;
```